spades/3.15.4/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:focal
```

Comment:
I'm not going to lie. I'm going to grab this file and see if I can create a test for it. 

Thank you for being an involuntary guinea pig.

Reply:
go right ahead 😁